docs/development/testguide.rst

```diff
@@ -764,66 +764,47 @@ Running image tests
 We make use of the `pytest-mpl <https://pypi.org/project/pytest-mpl>`_
 plugin to write tests where we can compare the output of plotting commands
 with reference files on a pixel-by-pixel basis (this is used for instance in
-:ref:`astropy.visualization.wcsaxes <wcsaxes>`).
+:ref:`astropy.visualization.wcsaxes <wcsaxes>`). We use the `hybrid mode
+<https://github.com/matplotlib/pytest-mpl#hybrid-mode-hashes-and-images>`_ with
+hashes and images.
 
-To run the Astropy tests with the image comparison, use::
+To run the Astropy tests with the image comparison, use e.g.::
 
-    pytest --mpl --remote-data=astropy
+    tox -e py39-test-image-mpl311
 
-However, note that the output can be very sensitive to the version of Matplotlib
-as well as all its dependencies (e.g., freetype), so we recommend running the
-image tests inside a `Docker <https://www.docker.com/>`__ container which has a
-frozen set of package versions (Docker containers can be thought of as mini
-virtual machines). See our ``.circleci/config.yml`` for reference.
+However, note that the output can be very sensitive to the specific version of
```

Comment:
As I pointed out earlier, in my experience (and in theory) when run with tox the hashes work 100% of the time on a linux machine.

I think we should explicitly state that the hashes only work on linux here if nothing else.

Also: it is sensitive to freetype, but if you install with tox you are always going to get the same freetype version mpl uses for that version, it's only if you are using system packages or another odd env that you would run into an issue.

Reply:
I've clarified this